packages/frontend/src/components/FavoriteSelector.vue

```diff
@@ -11,7 +11,7 @@
         />
       </div>
       <div v-for="task in tasks" :key="task.id" class="row">
-        <TimeEntrieText :task="task" />
+        <TimeEntrieText :task="task" :activityView="true" />
```

Comment:
Personlig syntes jeg det var lettere å få med seg at den var locked når hengelåsen var rett til venstere for checkboxen i activityview. Så for min del kan du bare fjerne activityView logikken igjen :)

Reply:
Fixed